### ISSUE TITLE
YOTHatch cant be crafted because of missing ME Fluid Interface

### ISSUE THREAD

**Issue by Ngar1** [GTNewHorizons/GT-New-Horizons-Modpack#10063]
### Your GTNH Discord Username

Ngar

### Your Pack Version

1.7.10-2.1.2.3qf

### Your Server

MyFTB

### Your Expectation

![grafik](https://user-images.githubusercontent.com/7759876/160547765-0a2b4aad-8791-4c8e-9003-bc06bab690d1.png)

### The Reality

no receipt for ME Dluid Interface

### Your Proposal

Fix

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [ ] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by minecraft7771** [comment]
I don't think this is an issue. The hatch is designed for use with extra cells 2 fluid integration, which is going obsolete, hence the missing recipe for the fluid interface. So it wouldn't make sense to add an alternative recipe for it.

**Comment by Dream-Master** [comment]
@GlodBlock want to hear you on this.
If it is possible a fixed ec2 mod would be better than a complete disabled one.

**Comment by GlodBlock** [comment]
that recipe has been changed by https://github.com/GTNewHorizons/GoodGenerator/pull/20, now it is using normal me interface.
but anyway it doesn't have much meaning if all ec2 parts disabled.

**Comment by Dream-Master** [comment]
@Ngar1 @GlodBlock can we close this issue as fixed in dev ?
